action-custom.py

```diff
@@ -12,7 +12,18 @@
 
 HERMES_HOST = "localhost"
 SECONDS_IN_A_DAY = 86400.0
-ARTIST_ENTITY_NAME = "snips/artist"
+
+ENTITIES = {
+    "artists": "snips/artist",
+    "tracks": "snips/song",
+    "playlists": "playlistNameFR",
+}
+
+# Logging config
+if HERMES_HOST != "localhost":
```

Comment:
It might be a good idea to have a flag outside of the script to decide of the logging level (sys.argv)